### PR TITLE
Fixes #505

### LINKED ISSUE #505: Fix typo in validation.adoc
_The `dolphin-platform-bean-validation` module don't depend ..._ should read: _The `dolphin-platform-bean-validation` module **doesn't** depend ..._

### PR DESCRIPTION
**Pull request by zimmi**
Fixes a minor typo in the documentation.

---
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/507)

### CODE REVIEW

**Comment by ameissne** [comment]
Great work!

File: documentation/src/docs/asciidoc/validation.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ adds a transitive dependency to the core module.
 .Dependencies for the validation module
 image::validation-dependency.png[]
 
-The `dolphin-platform-bean-validation` module don't depend on a JSR-303 implementation. If your application server don't
+The `dolphin-platform-bean-validation` module doesn't depend on a JSR-303 implementation. If your application server don't
```
Comment (ameissne) [inline documentation/src/docs/asciidoc/validation.adoc]: Excelent

**Comment by hendrikebbers** [comment]
@zimmi please link issue :)

**Comment by zimmi** [comment]
This fixes: https://github.com/canoo/dolphin-platform/issues/505

**Comment by hendrikebbers** [comment]
lgtm

**Pull request merged**